multiraft/multiraft.go

```diff
@@ -341,6 +354,42 @@ func (s *state) start() {
 			s.stop()
 			return
 
+		case req := <-s.reqChan:
+			_, groupOK := s.groups[req.GroupID]
+			_, nodeOK := s.nodes[req.Message.From]
+			groupOK = groupOK || req.Message.Type == raftpb.MsgHeartbeat
+			if !groupOK || !nodeOK || req.Message.To != s.nodeID {
+				log.V(4).Infof("node %v: discarding ill-addressed message %s", s.nodeID, req.GroupID,
+					s.EntryFormatter.DescribeMessage(req.Message))
+				break
+			}
+			log.V(5).Infof("node %v: group %v got message %s", s.nodeID, req.GroupID,
+				s.EntryFormatter.DescribeMessage(req.Message))
+			if req.Message.Type == raftpb.MsgHeartbeat {
+				// A heartbeat message is expanded into a heartbeat for each group
+				// that the remote node is a part of.
+				originNode, _ := s.nodes[req.Message.From]
```

Comment:
You don't need `, _` for a map lookup - just omit the second return value if you don't need it.